### PR TITLE
[fix] #2638: Update white paper.

### PR DESCRIPTION
**Pull request by SamHSmith**
Signed-off-by: Sam H. Smith <sam.henning.smith@protonmail.com>

### CODE REVIEW

File: docs/source/iroha_2_whitepaper.md [hunk 1]
```diff
@@ -262,47 +260,45 @@ to which it then will add its vote in order to get 2*f*+1 votes to commit the bl
...
-  - the *commit timer* on nodes will go off and a new leader and proxy tail are elected
+  - the *commit timer* on nodes will go off and a new leader and proxy tail are elected.
 
 - proxy tail does not inform any other node about a block commit (block withholding attack)
 
-  - the *commit timer* on nodes will go off and a new leader and proxy tail are elected;
-  the signatures from at least *f*+1 nodes saying their *commit timer* goes off, invalidates a block hash forever;
-  this invalidation is written in the next block created successfully, to prevent arbitrary rewriting of history in the future
+  - the *commit timer* on nodes will go off and a new leader and proxy tail are elected. Once 2 blocks have passed
+  arbitrary rewriting of history in the future is not possible. Once a subsequent block is produced it is not possible
```
Comment (outoftardis) [inline docs/source/iroha_2_whitepaper.md:289]:
```suggestion
  arbitrary rewriting of history in the future is not possible. Once a subsequent block is produced, it is not possible
```

File: docs/source/iroha_2_whitepaper.md [hunk 2]
```diff
@@ -262,47 +260,45 @@ to which it then will add its vote in order to get 2*f*+1 votes to commit the bl
...
 
   - through normal data synchronization (P2P gossip), set *b* will get up to date
 
 - proxy tail selectively sends a committed block to some, but not other nodes
 
-  - the *commit timer* on nodes will go off and a new leader and proxy tail are elected;
-  the signatures from at least *f*+1 nodes saying their *commit timer* goes off, invalidates a block hash forever;
-  this invalidation is written in the next block created successfully, to prevent arbitrary rewriting of history in the future
+  - Either the *commit timer* on nodes will go off and a new leader and proxy tail are elected, or the other nodes
+  will recieve the block through P2P sync. If a new round is performed the leader will add information to the new block
```
Comment (outoftardis) [inline docs/source/iroha_2_whitepaper.md:299]:
```suggestion
  will recieve the block through P2P sync. If a new round is performed, the leader will add information to the new block
```

File: docs/source/iroha_2_whitepaper.md [hunk 3]
```diff
@@ -315,14 +311,9 @@ to which it then will add its vote in order to get 2*f*+1 votes to commit the bl
...
-- a peer may make a false claim that their *voting timer* went off
-
-  - *f*+1 *voting timer* claims are required to make a block invalid and change the leader and proxy tail
-
-- a peer may make a leader suspect claim
-
-  - *f*+1 claims are needed to change a leader, so just one node is not enough; non-faulty nodes will not make false claims
+- a peer may make falsely suspect the *a* set
 
+  - *f*+1 suspicion claims are required to make a block invalid and change the leader and proxy tail
```
Comment (outoftardis) [inline docs/source/iroha_2_whitepaper.md:316]:
```suggestion
  - *f*+1 suspicion claims are required to make a block invalid and change the leader and the proxy tail
```

**Review by outoftardis — APPROVED** [review]

File: docs/source/iroha_2_whitepaper.md [hunk 4]
```diff
@@ -202,8 +202,7 @@ the submitting peer can use this as proof to convince non-faulty nodes to do a v
 
 - after a node signs off on a block and forwards it to the *proxy tail*,
 they expect a commit message within a reasonable amount of time (the *commit time*);
-if there is no commit message in time, the node tries to convince the network to do a view change,
-which creates a new leader and proxy tail by shifting down the 2*f*+1 nodes used by 1 to the right
+if there is no commit message in time, the node starts suspecting the *a* set.
```
Comment (appetrosyan) [inline docs/source/iroha_2_whitepaper.md:205]:
```suggestion
if there is no commit message in time, the node starts suspecting the $a$ set.
```

Comment (appetrosyan) [reply docs/source/iroha_2_whitepaper.md:205]: We're essentially differentiating between mathematical and non-mathematical usage of terms. Believe me, it'll render correctly.

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**